.github/workflows/eden_emh_arm.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: EdenEMH_ARM
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+  workflows:
+    - Publish
+  types:
+    - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        conf_eden_server: ["-l dfw2 -c c2.large.arm"]
+        conf_eve_server: ["-l dfw2 -c c2.large.arm"]
+        hv: ["kvm"]
+    steps:
+      - name: Setup packages
+        run: |
+          sudo rm -rf /etc/apt/sources.list.d/microsoft-prod.list /etc/apt/sources.list.d/microsoft-prod.list.save
+          sudo add-apt-repository ppa:longsleep/golang-backports
+          sudo apt install -y golang jq expect
+      - name: Get eden
+        uses: actions/checkout@v2
+      - name: Create ubuntu server on packet hosting
```

Comment:
In GCP we we have the opportunity to create network tunnel betwen GCP EVE and EDEN that running in workflow, but we have no way to do it with the packet. There is no such functionality on the packet side. Therefore, I did that a server with EDEN is also being created.

Reply:
In theory, we can create a virtual network in packet-hosting, add a ubuuntu-server with a DHCP-server (for a virtual network) and a VPN on an external IP to this network. After that, create EVE with this virtual network. And connect this setup to that virtual network via VPN. But this is in theory, do we need such difficulties? If necessary, I can do something similar.

Reply:
> In GCP we we have the opportunity to create network tunnel betwen GCP EVE and EDEN that running in workflow, but we have no way to do it with the packet.

I don't see why not. We have a VPN service for GCP. It is maintained by @mydatascience (btw have you spoken to him about this?). I see no reason not to use it, especially since the secret for it is already available to GH actions.

In fact, if you ask me, there's no reason why both GCP and Packet shouldn't share common codebase.

Reply:
@rvs Yes, I will definitely talk with Vladimir about this.

Reply:
The main problem here: inside GCP we have pre-defined route 10.8.0.0/24 via openvpn instance. So, when we create VM with EVE, it use this route to pass traffic to GH Action VM (which connects to openvpn). But in case of Packet, we cannot create routes for VMs.

Reply:
@giggsoff we can use another method for managing routes, but I need test this. The idea is to create a virtual local area network, to which add a server on which DHCP will be deployed, and this server will act as a gateway for this network. then install OpenSSH on this server and configure routes. Then you can simply add EVE to this virtual network. There are a couple of pitfalls with route priorities in EVE itself, but I think they can be solved.

Reply:
It's definitely easier to do it the same way as in GCP, but in packet instead of using GCP infrastructure for the packet (we need same nw not the tunnel). Adam in the cloud is a good option as well. We may look into this option.